src/include/btmem.h

```diff
@@ -827,8 +827,9 @@ struct __wt_page {
  */
 struct __wt_page_deleted {
 	volatile uint64_t txnid;		/* Transaction ID */
-	wt_timestamp_t timestamp;
-	wt_timestamp_t durable_timestamp;	/* aligned uint64_t timestamp */
+
+	wt_timestamp_t timestamp;		/* Timestamps */
```

Comment:
Are we not calling these start_ts, durable_ts like at other places.

Reply:
We're pretty evenly split between `XXX_timestamp` and `XXX_ts`. I've been just leaving it alone, once this ticket gets merged, I plan to do a pass and bring everything into alignment using `XXX_ts`.